### ISSUE TITLE
[TW-1305] Make tasks with due:someday less urgent than task without a due date

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1338]
_Tobias Mueller on 2014-04-15T13:26:59Z says:_ 

 When creating a task with due:someday it is considered more important than tasks without a due date set.

I think it is not intuitive as I set due:someday to indicate that I don't want to look at that task until I have some spare cycles. Until that is the case, I usually want to work on every other task, including those for which I did not set a due date.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2014-04-15T13:26:59Z
Modified: 2017-01-16T18:50:43Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2014-04-16T01:21:20Z says:_ 

 Thank you for adding this.  I would like anyone who agrees/disagrees to speak up, approve/disapprove this issue, and we can then move forward with a consensus.

**Comment by taskwarrior** [comment]
_Wim Schuermann on 2014-04-16T05:14:31Z says:_ 

 I agree with Tobias, and I currently use a workaround that feels wrong every time I use it: Instead of setting a due date, I set a +maybe / +someday tag.

The problem is that even this changes the task's urgency and raises it above tasks without any metadata, so it is not a perfect workaround either. It works for me because I don't consider tasks with more tags to be more important, so I can just do "task config urgency.tags.coefficient 0" and go on my merry way.

Being able to set the actual due date to "someday" and having that not affect the urgency (or, heck, lower it instead of making it higher) would feel right to me. It is "The GTD Way"(tm), and I don't see any downsides.

**Comment by xerus2000** [comment]
Yes, I would love this as well! Currently I tend towards setting low priorities and sometimes wait:someday, but these are rather half-assed workarounds.

**Comment by Andonome** [comment]
Perhaps no due date could default to some assumed due date, e.g. 1 year in  the future (relative to the real date), so tasks due 2 years in the future would receive a lower priority than tasks without an explicit due date, but after 1 year, they would begin to become more urgent than tasks without a due date.
